README.rst

```diff
@@ -39,6 +38,48 @@ a virtual environment):
     $ pip install -e .
 
 
+Building a Python ZIP using `shiv <http://shiv.readthedocs.io/en/latest/>`_
+---------------------------------------------------------------------------
+
+To create a .pyz with all dependencies included, you'll need to `install shiv <https://github.com/linkedin/shiv>`_
```

Comment:
please,
1) use named refrences like `CSR` at the bottom.
2) link only once to a resource regarding `shiv`, imo that should be the documentation.
3) never put hyperlinks into headings as they are usually a link target.

regarding the description:
- the reader at this point doesn't know what a `.pyz` is, s/he should be informed that `shiv` can be used to create Python virtual environments that contain a package and all its dependencies incl. binary libraries for a specific platform that can then be distributed and executed on systems that have a Python interpreter installed w/o further effort
- file extensions can change over time, so it's clearer to name things as what they are, eg. a Word document, not a `doc` or `docx`
- if i understood and observed correctly, the package that is to be packaged doesn't need to be installed, but its source must be locally available

as we intend to recommend this as primary way to get `quick-csr` up and running, i'd say this section should be above of the 'advanced' Installation section.